2015/proposals/agarwal-rajat.md

```diff
@@ -0,0 +1,92 @@
+# Title
+
+Add taxonomic name resolution to the EcoData Retriever to facilitate data science approaches to ecology
```

Comment:
I'd probably change "data science approaches to ecology" to "data-intensive ecology"